cmake/configure/configure_1_mpi.cmake

```diff
@@ -44,6 +44,13 @@ ENDMACRO()
 
 MACRO(FEATURE_MPI_CONFIGURE_EXTERNAL)
 
+  #
+  # We must convert the MPIEXEC_(PRE|POST)FLAGS strings to lists in order
+  # to use them in command lines:
+  #
+  SEPARATE_ARGUMENTS(MPIEXEC_PREFLAGS NATIVE_COMMAND "${MPIEXEC_PREFLAGS}")
+  SEPARATE_ARGUMENTS(MPIEXEC_POSTFLAGS NATIVE_COMMAND "${MPIEXEC_POSTFLAGS}")
```

Comment:
cmake 3.5 from the tester complains about `NATIVE_COMMAND` which is some newer cmake feature (I think 3.9). How about using the basic `SEPARATE_ARGUMENTS(MPIEXEC_{PRE,POST}FLAGS)` which should do what we want?

Reply:
Good point, changing.